### DISCUSSION TITLE
Aurora Global Database

### DISCUSSION THREAD

**Discussion by ChrisRut** [awslabs/aws-postgresql-jdbc#32] [Q&A] [answered]
Will this work with Aurora Global Database failovers ?

**Comment by samathaamz** [comment, marked as answer]
Hi @ChrisRut,

Thanks for reaching out and inquiring about the driver. The current version of the driver does not support Aurora Global Databases. To clarify, it does not support the global failover feature. However, we'd like to share some information regarding behaviour.

For Aurora's Global Databases, in terms of a failover procedure within a cluster (a downed instance), our current driver supports failover for the primary clusters. However, it does not support failover for any secondary clusters since they do not have writers.

Currently support for global databases is not on the roadmap for GA, but we'll share feedback with the team on the interest in supporting the feature.
